### ISSUE TITLE
Could you share the dependency you used for finetuning/train vidtok on mi300?

### ISSUE THREAD

**Issue by Zehui127** [microsoft/VidTok#21]
Dear Team,

Could you share the python packages with corresponding versions when running the vidtok on mi300x? Specifically, which rocm did you use? 

I have being encountering issues when running the code with 

torchrun --nnodes=$NODES --nproc_per_node=$GPUS main.py ... 

Best,
Zehui

**Comment by Zehui127** [comment]
for those who might want to run vidtok on mi300x, the following config work for me: 

rocm6.3.1, PyTorch 2.5,

for the other package:

pip install --user opencv-python-headless numpy decord tqdm omegaconf pytz

pip install --user absl-py==2.1.0 aiohappyeyeballs==2.4.4 aiohttp==3.11.9 aiosignal==1.3.1 antlr4-python3-runtime==4.9.3 appdirs==1.4.4 async-timeout==5.0.1 attrs==24.2.0 av==12.0.0 beartype==0.18.2 certifi==2024.8.30 charset-normalizer==3.4.0 click==8.1.7 contourpy

pip install --user cycler==0.12.1 decord==0.6.0 docker-pycreds==0.4.0 einops==0.8.0 filelock==3.16.1 fonttools==4.55.1 frozenlist==1.5.0 fsspec==2024.10.0 gitdb==4.0.11 gitpython==3.1.43 grpcio==1.68.1 idna==3.10 imageio==2.34.0 jinja2==3.1.4 kiwisolver==1.4.7 lightning==2.2.4

pip install --user lightning-utilities==0.11.9 markdown==3.7 markdown-it-py==3.0.0 markupsafe==3.0.2 matplotlib==3.8.4 mdurl==0.1.2 mpmath==1.3.0 multidict==6.1.0 natsort==8.4.0 networkx numpy

pip install --user omegaconf opencv-python packaging pandas pillow propcache protobuf psutil pygments pyparsing python-dateutil pytz pyyaml requests rich safetensors sentry-sdk setproctitle six smmap sympy tensorboard tensorboard-data-server torch torchmetrics torchvision tqdm triton typing-extensions tzdata urllib3 wandb werkzeug yarl torchmetrics "pytorch-lightning==2.2.4"
